### PR TITLE
feat: pdf argument

### PR DESCRIPTION
**Pull request by nyxordinal**
Changes:
* Add new argument `--pdf` to render the pdf file without having to interact with the `inputimeout` dialog
* Fix some wrong grammar in the `README.md` file 

Issue: #3

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,6 +24,9 @@ asmhentai:
 
 bulk:
 	tomoe --bulk doujin.json
+
+pdf:
```
Comment (sinkaroid) [inline Makefile:28]: I see a new target testing here, but no new workflows was added, If you want to add testing for pdf could you add the new `.github/workflows` ci too?

**Review by sinkaroid — CHANGES_REQUESTED** [review]
Hey thanks, everything looks good for me, It's all tested already? I have tested nhentai and pururin that was beautiful. 

Small changes here:
- Bump new release to: `__version__ = "3.0.14"` please?
- Could you apply [`black`](https://pypi.org/project/black/) on your commit? A code formatter like prettier then run `black tomoe/`

File: Makefile (hunk 1 above)
Comment (nyxordinal) [reply Makefile:28]: workflow added

**Comment by nyxordinal** [comment]
PR updated @sinkaroid

**Review by sinkaroid — APPROVED** [review]

**Pull request merged**